test/Mono.Linker.Tests.Cases/Logging/SourceLines.cs

```diff
@@ -8,27 +9,42 @@ namespace Mono.Linker.Tests.Cases.Logging
 	[SkipKeptItemsValidation]
 	[SetupCompileBefore ("FakeSystemAssembly.dll", new[] { "../PreserveDependencies/Dependencies/PreserveDependencyAttribute.cs" })]
 	[SetupCompileArgument ("/debug:full")]
-	[LogContains ("(30,4): Trim analysis warning IL2074")]
-	[LogContains ("(31,4): Trim analysis warning IL2074")]
+	[LogContains ("(33,4): Trim analysis warning IL2074")]
+	[LogContains ("(34,4): Trim analysis warning IL2074")]
+	[LogContains ("(38,3): Trim analysis warning IL2091")]
 	public class SourceLines
 	{
 		public static void Main ()
 		{
 			UnrecognizedReflectionPattern ();
+			GenericMethodIteratorWithRequirement<SourceLines> ();
 		}
 
 		[DynamicallyAccessedMembers (DynamicallyAccessedMemberTypes.PublicConstructors)]
-		private static Type type;
+		static Type type;
 
-		private static Type GetUnknownType ()
+		static Type GetUnknownType ()
```

Comment:
Was this change needed for line numbers to show up (should have same meaning in IL)?

Reply:
It's just style - in this repo the coding style is to not specify `private` because it's the default. I think for this one I simply ran `lint.cmd` and it changed it like that.